LibGit-Core.package/LGitRemote.class/instance/remote_connect.direction.callbacks..st

```diff
@@ -1,7 +1,9 @@
 libgit-calls
 remote_connect: remote direction: direction callbacks: remoteCallbacks
 	
-	^ self
-		ffiCallSafely:
-			#(LGitReturnCodeEnum git_remote_connect #(self , LGitDirectionEnum direction , LGitRemoteCallbacks * remoteCallbacks))
-		options: #()
\ No newline at end of file
+	^ self ffiLibrary uniqueInstance version first = 0
+		ifTrue: [
+			self shouldBeImplemented.
+			self remote_connect_v0251: remote direction: direction callbacks: remoteCallbacks ]
+		ifFalse: [ 
+			self remote_connect_v100: remote direction: direction callbacks: remoteCallbacks ]
```

Comment:
@tesonep @guillep @astares do we still need to support <v1.0.0 ?
I don't remember now details  but evidently I needed to add this version-dependent code... And I suspect we can assume >=v1.0.0.